Dockerfile

```diff
@@ -29,5 +28,7 @@ WORKDIR $HOME/dug
 
 RUN make install
 
+USER $USER
```

Comment:
Was running into permissions issues wrt `chown` during `docker-compose build` and moving the location resolved the issue.